### ISSUE TITLE
Configure other filetypes in EditorConfig

### ISSUE THREAD

**Issue by dhruvkb** [WordPress/openverse#779]
## Description

The `.editorconfig` file in the repo is incorrectly configured for the following file types.

- Python `*.py`
- Justfile `justfile`

^ Add more above.

The config needs to be extended to support the right formatting options for these files.

**Comment by muddi900** [comment]
Would this change be preferable:

```
[python]
charset = utf-8
indent_style = space
indent_size = 4
end_of_line = lf
insert_final_newline = true
trim_trailing_whitespace = true
```

**Comment by dhruvkb** [comment]
This has been fixed in 15e14a385ec1e1c42d740f0981558866bdeb5e28 by @dhruvkb and dc8b2f35541e3adfb787dcb1dd11a838dbafe1ea by @sarayourfriend.
